### PR TITLE
add automated lighthouse testing

### PR DESCRIPTION
**Pull request by goapunk**

### CODE REVIEW

File: lighthouse.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "pageName": "Landing Page",
+    "url": "http://localhost:8006"
+  },
+  {
+    "pageName": "Acadamy Landing Page",
```
Comment (goapunk) [inline lighthouse.json:7]: probably want more sites here

File: lighthouse.mjs [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import { join } from 'path'
+import { writeFileSync, mkdirSync, existsSync } from 'fs'
+import { launch } from 'chrome-launcher'
+import lighthouse from 'lighthouse'
+
+const urls = [
```
Comment (goapunk) [inline lighthouse.mjs:6]: more sites here? which ones?

File: lighthouse.mjs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  {
+    pageName: 'Landing Page',
+    url: ''
+  },
+  {
+    pageName: 'Acadamy Landing Page',
+    url: '/academy'
```
Comment (fuzzylogic2000) [inline lighthouse.mjs:13]: I think in the current setup, this url is the academy index page?

File: lighthouse.mjs (hunk 2 above)
Comment (fuzzylogic2000) [reply lighthouse.mjs:6]: yeah, how do we do it? Check the pages and add at least one example of all our pages manually here?

File: lighthouse.mjs (hunk 3 above)
Comment (goapunk) [reply lighthouse.mjs:13]: fixed

**Review by fuzzylogic2000 — COMMENTED** [review]
It does work! But the page naming might be a problem, the /academy seems not to exist on the stage server.

File: lighthouse.mjs (hunk 2 above)
Comment (goapunk) [reply lighthouse.mjs]: I guess, yeah. Can also be done over time I suppose

File: lighthouse.mjs [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+import lighthouse from 'lighthouse'
+
+const serverUrls = [
+  {
+    pageName: 'Landing Page',
+    url: ''
+  },
+  {
+    pageName: 'Acadamy Index Page',
+    url: '/academy'
```
Comment (fuzzylogic2000) [inline lighthouse.mjs:13]: liquid-academy on stage, I think!

**Review by fuzzylogic2000 — APPROVED** [review]
Cool!

**Pull request merged**